src/Sentry/SentryFeedback.cs

```diff
@@ -0,0 +1,101 @@
+using Sentry.Extensibility;
+using Sentry.Internal;
+using Sentry.Internal.Extensions;
+
+namespace Sentry;
+
+/// <summary>
+/// Sentry User Feedback.
+/// </summary>
+public sealed class SentryFeedback : ISentryJsonSerializable, ICloneable<SentryFeedback>
+{
+    /// <summary>
+    /// Tells Sentry which type of context this is.
+    /// </summary>
+    internal const string Type = "feedback";
+
+    /// <summary>
+    /// Message containing the user's feedback.
+    /// </summary>
+    public string Message { get; set; } = string.Empty;
+
+    /// <summary>
+    /// The name of the user.
+    /// </summary>
+    public string? ContactEmail { get; set; }
+
+    /// <summary>
+    /// The name of the user.
+    /// </summary>
+    public string? Name { get; set; }
+
+    /// <summary>
+    /// Optional ID of the Replay session associated with the feedback.
+    /// </summary>
+    public string? ReplayId { get; set; }
+
+    /// <summary>
+    /// The name of the user.
```

Comment:
Name of the user or a URL? What URL would it be?

afaik a feedback is an event. Should we just derive from it?